engine/apps/alerts/tasks/notify_team_members.py

```diff
@@ -0,0 +1,38 @@
+
+from django.conf import settings
+from django.db import transaction
+
+from common.custom_celery_tasks import shared_dedicated_queue_retry_task
+
+from .task_logger import task_logger
+from .notify_user import notify_user_task
+
+@shared_dedicated_queue_retry_task(
+    autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else None
+)
+def notify_team_members_task(
+    team_pk,
+    alert_group_pk,
+    previous_notification_policy_pk=None,
+    reason=None,
+    prevent_posting_to_thread=False,
+    notify_even_acknowledged=False,
+    important=False,
```

Comment:
I don't see the `important` flag to be set anywhere? I think you need to check the escalation step and set accordingly before queuing the task. Also, maybe re-check what params are used/needed here (I don't think we need them all?)

Reply:
Fixed